tuxemon/states/combat/combat.py

```diff
@@ -1013,7 +1017,15 @@ def perform_action(
                     )
                 )
 
-        tech_sprite = self._technique_cache.get(technique)
+        # TODO: Flipping in X-axis and Y-axis should be allowed,
+        #       and supported in the technique or animation JSON
+        flip_axes = ""
+        for trainer in self.ai_players:
+            if user in self.monsters_in_play[trainer]:
+                flip_axes = "x"
```

Comment:
This should be a bool that defines if we should show the normal animation or the flipped animation. The info on how to flip it should be on the technique object.

Reply:
Is the new implementation better?

Reply:
Yes